app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/UploaderQueue.java

```diff
@@ -146,7 +148,41 @@ public static UploaderQueue newEntry(String action, Model obj) {
         result.bitfield_wanted = DEFAULT_UPLOAD_CIRCUITS
                 | (Home.getPreferencesBooleanDefaultFalse("cloud_storage_mongodb_enable") ? MONGO_DIRECT : 0)
                 | (Home.getPreferencesBooleanDefaultFalse("cloud_storage_api_enable") ? NIGHTSCOUT_RESTAPI : 0)
-                | (Home.getPreferencesBooleanDefaultFalse("cloud_storage_influxdb_enable") ? INFLUXDB_RESTAPI : 0);
+                | (Home.getPreferencesBooleanDefaultFalse("cloud_storage_influxdb_enable") ? INFLUXDB_RESTAPI : 0)
```

Comment:
The way diff shows this really confused me as the changes are minimal in `newEntry()`